### ISSUE TITLE
Support working in serverless-offline environment

### ISSUE THREAD

**Issue by aldarund** [thundra-io/thundra-lambda-agent-nodejs#26]
I expect it not to crash handler when working in local emulation e.g. serverless-offline environment. It dont need to send any data or anything, it just should work as without thundra at all.
Right now it just crash right on import
```
'getconf' is not recognized as an internal or external command,
operable program or batch file.
Serverless: Error while loading nuxtRenderer
[ 'Error: Command failed: getconf CLK_TCK',
  '\'getconf\' is not recognized as an internal or external command,',
  'operable program or batch file.',
  '',
  'at checkExecSyncError (child_process.js:603:11)',
  'at Object.execSync (child_process.js:640:13)',
  'at new <anonymous> (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\@thundra\\core\\dist\\webpack:\\thundra\\src\\plugins\\Metric.ts:34:35)',
  'at Object.e.default (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\@thundra\\core\\dist\\webpack:\\thundra\\src\\plugins\\Metric.ts:150:12)',
  'at t.exports.t (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\@thundra\\core\\dist\\webpack:\\thundra\\src\\index.ts:32:30)',
  'at Object.<anonymous> (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\handler.js:8:41)',
  'at Module._compile (internal/modules/cjs/loader.js:689:30)',
  'at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)',
  'at Module.load (internal/modules/cjs/loader.js:599:32)',
  'at tryModuleLoad (internal/modules/cjs/loader.js:538:12)',
  'at Function.Module._load (internal/modules/cjs/loader.js:530:3)',
  'at Module.require (internal/modules/cjs/loader.js:637:17)',
  'at require (internal/modules/cjs/helpers.js:20:18)',
  'at Object.createHandler (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\serverless-offline\\src\\functionHelper.js:104:21)',
  'at handler (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\serverless-offline\\src\\index.js:548:40)',
  'at Object.internals.handler (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\handler.js:96:36)',
  'at request._protect.run (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\handler.js:30:23)',
  'at module.exports.internals.Protect.internals.Protect.run (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\protect.js:64:5)',
  'at exports.execute (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\handler.js:24:22)',
  'at each (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:384:16)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at internals.Auth.payload (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\auth.js:223:16)',
  'at each (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:384:16)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at internals.payload (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\route.js:393:16)',
  'at each (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:384:16)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at module.exports.internals.Auth.internals.Auth._authenticate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\auth.js:210:16)',
  'at internals.Auth.authenticate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\auth.js:202:17)',
  'at each (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:384:16)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at request.connection.states.parse (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\route.js:379:20)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:244:16)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:31:25)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:184:20)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:184:20)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:184:20)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:184:20)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at done (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:28:25)',
  'at Items.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:184:20)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at Object.exports.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:39:9)',
  'at exports.Definitions.internals.Definitions.internals.Definitions.parse (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\statehood\\lib\\index.js:158:11)',
  'at internals.state (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\route.js:360:31)',
  'at each (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:384:16)',
  'at iterate (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:36:13)',
  'at Object.exports.serial (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\items\\lib\\index.js:39:9)',
  'at internals.Request._lifecycle (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:387:11)',
  'at internals.Request._execute (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\request.js:302:21)',
  'at Domain.request._protect.enter (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\connection.js:261:25)',
  'at Domain.run (domain.js:343:14)',
  'at module.exports.internals.Protect.internals.Protect.enter (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\protect.js:80:17)',
  'at Server.<anonymous> (C:\\Dev\\PycharmProjects\\rasimplefront\\ra-front-simple-nu\\front\\node_modules\\hapi\\lib\\connection.js:259:30)',
  'at Server.emit (events.js:182:13)',
  'at Server.EventEmitter.emit (domain.js:442:20)',
  'at parserOnIncoming (_http_server.js:658:12)' ]
```

**Comment by Jayasagar** [comment]
I have same issue, unable to run using `serverless-offline` 

Exception

```
(node:56662) UnhandledPromiseRejectionWarning: TypeError: Cannot destructure property `threadCount` of 'undefined' or 'null'.
    at Object.<anonymous> (/Users/jay/codeandco/volkswagon/vw-api-service/node_modules/@thundra/core/dist/thundra.js:1:25526)
    at Generator.next (<anonymous>)
```

**Comment by kobalski** [comment]
Fixed by #32

**Comment by kobalski** [comment]
You can use `v2.0.1` of Thundra NodeJS agent with offline environment.
